src/main/scala/circt/stage/Shell.scala

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package circt.stage
+
+import chisel3.stage.{
+  ChiselCircuitAnnotation,
+  ChiselGeneratorAnnotation,
+  CircuitSerializationAnnotation,
+  PrintFullStackTraceAnnotation,
+  SourceRootAnnotation,
+  ThrowOnFirstErrorAnnotation,
+  UseLegacyShiftRightWidthBehavior,
+  WarningConfigurationAnnotation,
+  WarningConfigurationFileAnnotation,
+  WarningsAsErrorsAnnotation
+}
+import firrtl.options.BareShell
+import firrtl.options.TargetDirAnnotation
+import logger.{ClassLogLevelAnnotation, LogClassNamesAnnotation, LogFileAnnotation, LogLevelAnnotation}
+
+trait CLI { this: BareShell =>
+
+  /** Include options for logging
+    *
+    * Defaults to true, override to false when mixing in to [[firrtl.options.Shell]] for use in a [[firrtl.options.Phase]]
+    */
+  def includeLoggerOptions: Boolean = true
+
+  if (includeLoggerOptions) {
+    parser.note("Logger options")
+    Seq(LogLevelAnnotation, ClassLogLevelAnnotation, LogClassNamesAnnotation).foreach(_.addOptions(parser))
+  }
```

Comment:
This may be better as a `CLI` trait in the logger package that is mixed into the `Shell` as opposed to a method that can be overridden. If going with the method approach, this would be better as `protected`.

Reply:
It definitely should be protected, will fix. The reason it's a method is because this trait needs to get mixed in to the Shell used by _class_ ChiselStage, but that has to use `firrtl.options.Shell` (required as the type by `Stage`) which already includes the Logger options. So this way makes it _reasonably_ neat and re-usable by class and object ChiselStage.